### PR TITLE
[WEB][CHROME][CRITICAL] Adding new Test Case TC008

### PR DESCRIPTION
**Pull request by prazwal23**
# What is this PR for?
- To create a new Test Script for Test Case [TC008](https://www.google.com)

# Files Modified?
- NA

# Library Files Added/Modified?
- NA

# Files Newly Added?
- Skillkart/src/test/java/com/etrading/skillkart/scripts/TC008.java

# Jenkins Execution Result
- [Jenkins Success job](https://www.google.com)

### CODE REVIEW

File: Skillkart/src/test/java/com/etrading/skillkart/scripts/TC008.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.etrading.skillkart.scripts;
+import org.testng.annotations.Test;
+public class TC008 extends BaseTest
+{
+	@Test(description="for github")
+	public void dummyTestCase()
```
Comment (nameishemanthSCSW5) [inline Skillkart/src/test/java/com/etrading/skillkart/scripts/TC008.java]: This Code is not correct pls fix it

Comment (prazwal23) [reply Skillkart/src/test/java/com/etrading/skillkart/scripts/TC008.java]: I Have fixed pls review

Comment (nameishemanthSCSW5) [reply Skillkart/src/test/java/com/etrading/skillkart/scripts/TC008.java]: :+1:

**Review by nameishemanthSCSW5 — APPROVED** [review]
:+1: LGTM

**Pull request merged**